xyzprinting_tough_pla.xml.fdm_material

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
```

Comment:
If this isn't different from generic tough PLA, there isn't really a need to add this.

Reply:
ok, i got it, but the temperature is a little different.